webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBar.test.js

```diff
@@ -58,6 +58,7 @@ describe('BreadcrumbBar', () => {
     });
 
     it('onclick callbacks should work', () => {
+      window.history.pushState({}, 'Test Title', '/hosts/1');
```

Comment:
If you would use the `getCurrentPath` from the `LayoutHelper` you will be able to use mocks.

```js
import { getCurrentPath } from '../../../common/helpers';
jest.mock('../../../common/helpers');

it('onclick callbacks should work', () => {
  getCurrentPath.mockImplementation(() => '/hosts/1')
  ...
});
```